test/orange/orangeinp/CsgTreeUtils.test.cc

```diff
@@ -231,6 +240,167 @@ TEST_F(CsgTreeUtilsTest, postfix_simplify)
     }
 }
 
+TEST_F(CsgTreeUtilsTest, infix_simplify)
+{
+    // NOTE: mz = below = "false"
+    auto mz = this->insert_surface(PlaneZ{-1.0});
```

Comment:
Would it be possible to add a ASCII diagram or description of what this cell looks like (i.e. concentric cylinders)?